gdal/build.sh

```diff
@@ -1,7 +1,21 @@
 #!/bin/bash
+./configure --with-python --prefix=$PREFIX \
+--with-geos=$PREFIX/bin/geos-config \
+--with-static-proj4=$PREFIX \
+--with-hdf5=$PREFIX \
+--with-hdf4=$PREFIX \
+--with-netcdf=$PREFIX \
+--with-xerces=$PREFIX \
+--with-sqlite3=$PREFIX \
+--without-pam \
+--disable-rpath
```

Comment:
I am not totally familiar with exactly how patchelf is used, but my understanding was that it only seeked out and modified rpaths. Once I finally get this package building on our system, I'll be able to try relocating it to some of the test machines to see if it is truly relocatable.